src/components/Error/Error.tsx

```diff
@@ -0,0 +1,56 @@
+import classNames from 'classnames';
+import React from 'react';
+import { ErrorType } from '../../types/ErrorType';
+
+type Props = {
+  errorType: ErrorType;
+  hasError: boolean;
```

Comment:
All good, but you can just check is there errorType - if yes so error is present
```
 errorType: ErrorType;
  hasError: boolean;
```
->
 `errorType: ErrorType | null`